### ISSUE TITLE
[storage-resize-images] GCS bucket does not shown in dropdown list

### ISSUE THREAD

**Issue by htsuruo** [firebase/extensions#1609]
### [REQUIRED] Step 2: Describe your configuration

- Extension name: `storage-resize-images`
- Extension version: 0.1.37

### [REQUIRED] Step 3: Describe the problem

I would like to use this extensions for separated bucket from Firebase Storage(`[project_id.appspot.com`). However, the buckets of my project **other than `[project_id].appspot.com`** cannot be shown in dropdown list. 

![unnamed](https://github.com/firebase/extensions/assets/12729025/94b8b5d2-3a1c-41b4-ab4d-588ab1f55c7a)

I think that this problem is caused by user permission, so I attached a role Storage Admin (roles/storage.admin) to login user. But the result did not change.

The roles of login user is here: 
<img width="258" alt="image" src="https://github.com/firebase/extensions/assets/12729025/b120d81d-723d-46cb-9bc4-3504c47dff0c">

What is the cause for the issue ?

---

Actually, the dropdown list is set by `selectResource` type in `extensions.yaml` file.
```yaml
type: selectResource
resourceType: storage.googleapis.com/Bucket
```

As you see below `firebase-tools` source code comment, the internal logic calls [bucket/list ](https://cloud.google.com/storage/docs/json_api/v1/buckets/list) API endpoint simply.
https://github.com/firebase/firebase-tools/blob/c0a59a59ca9230a98713553a1c0335d6957be154/src/gcp/storage.ts#L244C9-L244C71

In this [Try this method](https://cloud.google.com/storage/docs/json_api/v1/buckets/list), the reponse returns all the GCS bucket expectedly. 
Why is the different behavior this and dropdown list in configuration parameter?

#### Steps to reproduce:

1. Create a bucket in Google Cloud Console
1. Install `storage-resize-images` extension
1. Set up configuration parameters ← the issue occurs here

##### Expected result

All the GCS bucket can be shown in dropdown list and selectable.

Thanks.

**Comment by dackers86** [comment]
Hi @HTsuruo 

If you inspect the network request in the browser, does the correct data appear? I want to first check that this is not a UI issue.

**Comment by dackers86** [comment]
Hi @htsuruo 

Could this be an issue with the locations of the other buckets. For example, do they live in the same location as your current project?

**Comment by htsuruo** [comment]
@dackers86 

Thanks for reply. It seems that locations does not cause because other buckets are same location.

But, I tried to re-install this extension today, and then other buckets are shown expectedly.
Hmm, I don't know what causes this problem, but the issue was solved.  

<img width="386" alt="SCR-20230903-tnuk-2" src="https://github.com/firebase/extensions/assets/12729025/71ec3d32-cf24-4654-83ad-3875f35a732c">

So, close this issue, thanks.
